azure-pipelines.yml

```diff
@@ -44,6 +46,12 @@ steps:
 - task: NodeTool@0
   inputs:
     versionSpec: '16.x'
+# variables['Build.SourceVersionMessage'] is only accessible at step level, so we use it directly for condition here
```

Comment:
As we discussed and decided we don't need to have a separate command for hotfix releases
Dist tag should be updated for previous versions here, to use _hotfix_, otherwise - canary or specific verX for next releases: https://github.com/Sitecore/jss/pull/1328/files#diff-7915b9b726a397ae7ba6af7b9703633d21c031ebf21682f3ee7e6a4ec52837a5R129